packages/instant/.production.discharge.json

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "instant.0xproject.com",
+    "build_command": "yarn build",
```

Comment:
I think we should change add  `--env.production` but I can do this when I update my heap PR ( see staging and dogfood env setting here: https://github.com/0xProject/0x-monorepo/pull/1295/files )